### PR TITLE
merge-flow: add manual instruction for assets and rh-manifest.txt

### PR DESCRIPTION
**Pull request by jan--f**
The asset generation and add steps are taken from the worklfows `asset-cmd` input.
The rh-manifest.txt steps are copied from the actual step. That not ideal, as they can diverge, though I don't expect that to happen very often.

### CODE REVIEW

**Comment by jan--f** [comment]
We could make the rh-manifest step an input with a default value. That way we can reuse the commands in the manual instructions, but only have them defined in one place. Not sure if we'll ever have the need to customize the actual commands though.

File: .github/workflows/merge-flow.yaml [hunk 1]
```diff
@@ -182,6 +182,12 @@ jobs:
             fi
             go mod tidy
             go mod vendor
+            ${{ inputs.assets-cmd }}
+            if [ -f scripts/rh-manifest.sh ]; then
+              bash scripts/rh-manifest.sh
+              git add rh-manifest.txt
+              git diff --cached --exit-code || git commit -s -m "[bot] update rh-manifest.txt"
```
Comment (danielmellado) [inline .github/workflows/merge-flow.yaml:189]: so basically the idea here is to avoid committing if there's no diff to generate, isn't it?

Comment (jan--f) [reply .github/workflows/merge-flow.yaml:189]: yep, the commit only needs to me made if `rh-manifest.txt` actually changed. But note that this is only a descruption that gets posted to a PR in some cases. So a human actually executes the steps.

**Review by raptorsun — APPROVED** [review]
/lgtm

**Pull request merged**